walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -81,14 +83,19 @@ public void noOptionsForTooLongOfARequest() {
   @Test
   public void eventSplitsRestriction() {
     // The event should split the day into two options (before and after the event).
-    Collection<Event> events = Arrays.asList(new Event("Event 1",
-        TimeRange.fromStartDuration(TIME_0830AM, DURATION_30_MINUTES), Arrays.asList(PERSON_A)));
```

Comment:
Hmm so the formatting changes are great, but they are not related to the topic of this PR. Would you be able to make a separate chained PR just for that last commit (formatting FindMeetingQueryTest.java) and keep this one here with just the extra tests. Not sure if this is a reasonable request, but it would help maintain the focus of the PR better.

Reply:
Yea you're right about it not being relevant in the context of this PR. It would have been nice if I set up the repository to format upon merging but I haven't looked into that yet. For this PR, I'll keep it in, but next time I'll either isolate formatting or set up formatting upon merging to master.

Reply:
sounds good!